packages/ember-cli-eyeglass/README.md

```diff
@@ -45,6 +116,43 @@ const app = new EmberEngine(defaults, {
   }
 });
 ```
+ 
+Given the following folder structure:
+
+```sh
+
+my-engine/
+         /app/styles/
+                    /my-engine.scss
+                    /my-other-file.scss
+                    /_shared.scss
+         /addon/styles/
+                      /_shared.scss
+                      /my-engine.scss
+                      /secondary.scss             
+```
+
+If this is an eager engine:
+ 
+
+The contents of `my-engine/app/styles/my-engine.scss` will become `dist/assets/eager.css`
```

Comment:
it's named `eager.css`? does that file contain the css of all eager addons?

Reply:
ah, it should be called `my-engine.css` my example i was working from had it's eager engine named `eager` . Let me correct